skia/RSkSVGContainer.cpp

```diff
@@ -154,23 +129,27 @@ void RSkSVGContainer::setRoot(RSkSVGNode * rootNode) {
   }
 
   for (int i = 0; i < childRSkNodeList_.size(); ++i) {
-    RNS_LOG_MODE_FOR_CONTAINER(" setRoot for Child  with Tag : "<<(int)childRSkNodeList_[i]->tag());
-    auto node=static_cast<RSkSVGNode *>(childRSkNodeList_[i].get());
-    if(node) {
-     node->setRoot(rootNode);
+    if(childRSkNodeList_[i] != nullptr) {
```

Comment:
In what case this list can have a  null child

Reply:
Unsupported SVG components will be rendered as unimplemented views, which, when mounted with an SVG parent, will be stored as null instead of being dropped to maintain vector position. If they are not stored properly, an index mismatch may occur upon unmounting.

Another feasible option to maintain the index is to use a map of the index against an object, in which there is no need to store a null object for an unsupported component. It can be simply ignored, and the implementation can be made more robust by ensuring that only valid objects are stored.